src/ResetPasswordHelperInterface.php

```diff
@@ -28,7 +28,7 @@ interface ResetPasswordHelperInterface
      * and removeResetRequest() can eventually invalidate it by removing it
      * from storage.
      *
-     * @param ?int $resetRequestLifetime Override the default (to be added to interface in 2.0)
+     * param ?int $resetRequestLifetime Override the default (to be added to interface in 2.0)
```

Comment:
Because this argument is commented out in the method signature - PHP CS Fixer removes it. I dropped `@` to workaround it

Reply:
Hmm, that makes sense. But now users won't get any notification in their editor about this. This method comes from the Symfony docs - https://symfony.com/doc/current/contributing/code/bc.html#adding-an-argument-to-a-public-method

Is there a way we can supress this from php-cs-fixer?

Reply:
@weaverryan We can't skip specific lines of code in PHP CS Fixer - we either should skip the whole file or skip the rule. I updated this PR skipping the `no_superfluous_phpdoc_tags` rule completely for this project.